src/components/WebexSettings/WebexSettings.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
+  Tab,
+  TabContent,
+  TabList,
+  TabPane,
+  Tabs,
+} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Settings component
+ *
+ * @param {string} props.className Custom CSS class to apply
+ * @param {string} props.meetingID ID of the meeting to control
```

Comment:
"Meeting id" is enough, or, if you prefer, something like "Id of the meeting to set the preferred devices for"